<?xml version="1.0" ?>

<launch>
  <arg name="mode" value="monitor" />
  <include file="$(find franka_motion_control)/launch/tracking_controller_node.launch" pass_all_args="true"/>

  <!-- <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"> -->
      <!-- <remap from="robot_description" to="my_robot_description" /> -->
      <!-- <remap from="joint_states" to="/joint_pos_controller/joint_states" /> -->
      <!-- <param name="publish_frequency" value="1000" /> -->
  <!-- </node> -->

</launch>
